osh/builtin_comp.py

```diff
@@ -26,15 +27,17 @@
   from osh.split import SplitContext
   from osh.word_eval import NormalWordEvaluator
 
+
+HELP_TOPICS = [] # type: List[str]
+
 from mycpp import mylib
 if mylib.PYTHON:
   # Hack because we don't want libcmark.so dependency for build/dev.sh minimal
   try:
     from _devbuild.gen import help_
+    HELP_TOPICS = help_.TOPICS
```

Comment:
Ah, this is gonna require some more work. CI doesn't produce `_devbuild.gen.help_`. I forgot I manually generated this in my local tree when I first wrote this...

Reply:
Ah OK let me know if you want me to look at the `help` stuff tomorrow ... I can probably remember how it works :)

Reply:
That would be great!